README.md

```diff
@@ -32,7 +32,7 @@ Refer to README.md under its directory
 Pre-requisites
 --------------
 - Android N device, API >= 24
-- Android Studio 2.2 beta1 or better
+- Android Studio 3.0 better (this is a must in order to build the sample here)
 - Android NDK
     * [NDK-r12](https://github.com/android-ndk/ndk/wiki), compile as is
```

Comment:
Users would simply get the NDK through the SDK manager, no? It may be easier to point users here for the NDK components: https://developer.android.com/studio/projects/add-native-code.html#download-ndk

Reply:
changed.